<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlatButtonSettings</class>
 <widget class="QWidget" name="FlatButtonSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="layout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="2" column="1">
    <widget class="QLabel" name="hoverStyleLabel">
     <property name="text">
      <string>Hover style</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="transparentCheckBox">
     <property name="text">
      <string>Transparent</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QComboBox" name="rippleStyleComboBox">
     <item>
      <property name="text">
       <string>Centered</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Positioned</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>No Ripple</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="textAlignmentComboBox">
     <item>
      <property name="text">
       <string>Left</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Center</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QCheckBox" name="iconCheckBox">
     <property name="text">
      <string>Icon</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="buttonRoleComboBox">
     <item>
      <property name="text">
       <string>Default</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Primary</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Secondary</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QDoubleSpinBox" name="overlayOpacityDoubleSpinBox">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="checkableCheckBox">
     <property name="text">
      <string>Checkable</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="showHaloCheckBox">
     <property name="text">
      <string>Show halo</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="rippleStyleLabel">
     <property name="text">
      <string>Ripple style</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="overlayOpacityLabel">
     <property name="text">
      <string>Overlay opacity</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="disabledCheckBox">
     <property name="text">
      <string>Disabled</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="checkedCheckBox">
     <property name="text">
      <string>Checked</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="cornerRadiusLabel">
     <property name="text">
      <string>Corner radius</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="iconPlacementComboBox">
     <item>
      <property name="text">
       <string>Left</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Right</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QDoubleSpinBox" name="fontSizeDoubleSpinBox"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="buttonRoleLabel">
     <property name="text">
      <string>Button role</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QSpinBox" name="iconSizeSpinBox"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="iconPlacementLabel">
     <property name="text">
      <string>Icon placement</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="hoverStyleComboBox">
     <item>
      <property name="text">
       <string>No Overlay</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Tinted Overlay</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gray Overlay</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QSpinBox" name="cornerRadiusSpinBox"/>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="textAlignmentLabel">
     <property name="text">
      <string>Text alignment</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="iconSizeLabel">
     <property name="text">
      <string>Icon size</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="fontSizeLabel">
     <property name="text">
      <string>Font size</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="buttonTextLabel">
     <property name="text">
      <string>Button text</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLineEdit" name="buttonTextLineEdit"/>
   </item>
   <item row="10" column="0" colspan="3">
    <layout class="QVBoxLayout" name="layMain">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <item>
      <layout class="QHBoxLayout" name="layPresets">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="defaultPresetPushButton">
         <property name="text">
          <string>Apply default preset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="checkablePresetPushButton">
         <property name="text">
          <string>Apply checkable preset</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="11" column="0" colspan="3">
    <layout class="QHBoxLayout" name="layFlatBtn">
     <item>
      <spacer name="hspFlatBtnL">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="MaterialFlatButton" name="flatBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>I'm flat</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hspFlatBtnR">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="12" column="0" colspan="3">
    <spacer name="vspMain">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaterialFlatButton</class>
   <extends>QPushButton</extends>
   <header location="global">QtMaterialWidgets/flatbutton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
